client/src/Tools/_framework/ToolPanels/GeneralContentControls.tsx

```diff
@@ -397,6 +429,125 @@ export function GeneralContentControls({
         <input type="hidden" name="imagePath" value={imagePath ?? undefined} />
         <input type="hidden" name="_action" value="update general" />
         <input type="hidden" name="id" value={id} />
+
+        {!contentData.isFolder ? (
+          <FormControl>
+            <Flex flexDirection="column" width="100%" rowGap={6}>
+              <FormLabel mt="16px">Content Classifications</FormLabel>
+
+              <Accordion allowMultiple>
+                {contentData.classifications.map((classification, i) => (
+                  <AccordionItem key={`classification${i}`}>
+                    <HStack>
+                      <h2>
+                        <AccordionButton>
+                          <HStack flex="1" textAlign="left" direction={"row"}>
+                            ;<Text as="b">{classification.code}</Text>
+                            <Text fontSize={"small"} pt="2px">
+                              {classification.system.name}
+                            </Text>
+                          </HStack>
+                          <AccordionIcon marginLeft="7px" />
+                        </AccordionButton>
+                      </h2>
+                      <Spacer />
+                      <CloseButton
+                        onClick={() => {
+                          fetcher.submit(
+                            {
+                              _action: "remove content classification",
+                              activityId: id,
+                              classificationId: classification.id,
+                            },
+                            { method: "post" },
+                          );
+                        }}
+                      />
+                    </HStack>
+                    <AccordionPanel>
+                      <Text as="b">Category: </Text>
+                      <Text>{classification.category}</Text>
+                      <Text as="b">Description: </Text>
+                      <Text>{classification.description}</Text>
+                    </AccordionPanel>
+                  </AccordionItem>
+                ))}
+              </Accordion>
+
+              <AsyncSelect
```

Comment:
Since it takes a bit for the classification to appear, you can add a nice touch of a `<Spinner>` to the right of the `<AsyncSelect>`. It can be revealed just before submitting the "add classification" action and then hidden again when the change actually occurs with a `useEffect` triggered on `[contentData]`.

I also recommend putting a hidden `<Spinner>` to the right of each `<AccordianItem>` that is revealed when you click the close button and hidden again by the same `useEffect`.